cluster/src/main/java/com/hartwig/pipeline/tertiary/virusbreakend/VirusBreakendCommand.java

```diff
@@ -0,0 +1,37 @@
+package com.hartwig.pipeline.tertiary.virusbreakend;
+
+import com.hartwig.pipeline.calling.command.VersionedToolCommand;
+import com.hartwig.pipeline.execution.vm.BashCommand;
+import com.hartwig.pipeline.execution.vm.OutputFile;
+import com.hartwig.pipeline.execution.vm.VmDirectories;
+import com.hartwig.pipeline.resource.ResourceFiles;
+import com.hartwig.pipeline.tools.Versions;
+import static com.hartwig.pipeline.resource.ResourceNames.VIRUSBREAKEND_DB;
+
+public class VirusBreakendCommand implements BashCommand {
```

Comment:
VirusBreakendCommand can `extend` VersionToolCommand. Then you just pass the arguments to the super constructor. See `ChordExtractSigPredictHRD` for an example

Reply:
Much cleaner indeed, thanks.